lib/ansible/modules/cloud/azure/azure_rm_subnet_facts.py

```diff
@@ -115,11 +116,16 @@
                 service:
                     description:
                         - The type of the endpoint service.
+                    returned: always
+                    type: str
                     required: True
+                    sample: Microsoft.Sql
```

Comment:
We can merge this PR without solving this issue, but I notice that the two modules being updated here don't use the same term in the RETURN documentation - one uses `example` and the other one uses `sample`. We should use one or the other consistently.

Reply:
Thanks for your review! It's my fault that I didn't check it out. I should change 'example'  to 'sample', which has been changed.